docs/SupportedCommands/Commands.md

```diff
@@ -50,27 +50,33 @@
 
 | Proxied Command (Android only)    |
 |-----------------------------------|
-| `getNetworkConnection`<sup>2</sup>|
+| `getNetworkConnection`            |
 | `isAppInstalled`                  |
 | `isLocked`                        |
 | `longPressKeyCode`                |
 | `pressKeyCode`                    |
-| `setNetworkConnection`<sup>2</sup>|
+| `setNetworkConnection`            |
 | `toggleLocationServices`          |
 | `unlock`                          |
 
-
+### Attributes
 The following attributes can be queried using `attribute`
-|          Attribute           |          Engine Version Support          |
+
+Attributes can be appended to the strategy value with following format: `[@attributeType='attributeValue']`
+
+|          Attribute         |          Engine Version Support          |
```

Comment:
Having a `ReturnType` column would be helpful  since now we support multiple return types set by the tag. Not everyone will have access to the engine code